app/_locales/fr/messages.json

```diff
@@ -253,21 +430,48 @@
   "gasPriceRequired": {
     "message": "Prix du gaz requis"
   },
+  "generatingTransaction": {
+    "message": "Préparation de la transaction"
+  },
   "getEther": {
     "message": "Obtenir des Ether"
   },
   "getEtherFromFaucet": {
     "message": "Obtenir de l'Ether d'une faucet pour $1",
     "description": "Affiche le nom du réseau pour la faucet d'Ether"
   },
+  "getHelp": {
+    "message": "Obtenir de l'aide."
+  },
   "greaterThanMin": {
     "message": "doit être supérieur ou égal à $1.",
     "description": "helper pour la saisie hexadécimale en entrée décimale"
   },
+  "hardware": {
+    "message": "hardware"
+  },
+  "hardwareWalletConnected": {
+    "message": "Portefeuille Hardware connecté"
+  },
+  "hardwareWallets": {
+    "message": "Connecter un portefeuille hardware"
+  },
+  "hardwareWalletsMsg": {
+    "message": "Selectionnez le portefeuille hardware que vous voulez utiliser avec MetaMask"
+  },
+  "havingTroubleConnecting": {
+    "message": "Un problème de connection ?"
+  },
   "here": {
     "message": "ici",
     "description": "comme dans -cliquer ici- pour plus d'informations (en rapport avec troubleTokenBalances)"
   },
+  "hereList": {
+    "message": "Voici une liste !!!!"
```

Comment:
Nit: we don't use this one so we can probably leave it out. (I don't think it's worth checking each individually in this PR, I only checked this one because of the generous use of exclamation marks. 😄)

Reply:
I left it in since it's in the English locale also